src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -1,57 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Please don't delete this file again. It's why it disappeared from the repo last time. Sync your branch with that PR that's out to add this back in (or with master if that's already merged)

Reply:
I am really sorry about that! We were usually doing those things and as we were merging they got restored but now I see that I have not been doing proper practice. So sorry!